content/en/blog/2021-03-08-flux-is-a-cncf-incubation-project.md

```diff
@@ -0,0 +1,108 @@
+---
+author: dholbach
+date: 2021-03-10 06:30:00+00:00
+title: Flux is a CNCF Incubation project
+description: The Cloud Native Computing Foundation promoted Flux to Incubation status. We celebrate all the great work that turned Flux into a GitOps family of projects.
+url: /blog/2021/03/flux-is-a-cncf-incubation-project/
+---
+
+![Incubation](/img/incubation.png)
+
+The Flux community is proud to announce that the CNCF Technical
+Oversight Committee just promoted the Flux project to Incubating status!
+🎉
+
+For us as a project, this is proof not only of the wide-spread use of
+Flux (and Helm Operator) v1, but also of our big strides into becoming a
+GitOps family of projects and bringing a toolkit approach to Continuous
+and Progressive Delivery.
+
+We are particularly happy about the support from our community, the
+project maintainers, contributors and end-users who helped us through
+this process. Thanks to our friends at Deutsche Telekom, Lunar,
+MediaMarktSaturn and Sortlist who spoke in favour of Flux and explained
+their GitOps implementation in user interviews with the CNCF.
+
+![Incubation Infographics](/img/incubation-infographics.png)
```

Comment:
Can we center this image? Here's an inline suggestion below

```suggestion
![Incubation Infographics](/img/incubation-infographics.png#center)
```

which - if it still works - should just need a CSS addition: https://discourse.gohugo.io/t/handling-images-size-aligning/1940/2

Reply:
I'm off for today and will be off tomorrow too. No time to check CSS additions. :man_shrugging:  Seperate PR?